docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,47 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. Note that because these metrics are calculated using 
```

Comment:
Clarification: there's ratio/diff that is max/min or max-min, but some people prefer relative to mean. That's also possible with MetricFrame.

Reply:
I can add this distinction as a new sentence (that the radio/diff can be max/min or max-min and that MetricFrame also provides this capability), but let me know if there is better phrasing I can use to distinguish between the two.

Reply:
Added a sentence: "Either the ratio or difference can be calculated as the max, min or max-min between subgroups (:class:`MetricFrame` also has the capability to calculate these metrics)."